pages/articleDetails/[id]/index.js

```diff
@@ -0,0 +1,87 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import { fetcher } from "../../../helpers/api";
+import Image from "next/image";
+import styled from "styled-components";
+import css from "styled-jsx/css";
+import Svg from "../../../components/Svg";
+import Link from "next/link";
+
+function articleDetails() {
+  const { query } = useRouter();
+  const { id } = query;
+  const { data: article, error } = useSWR(`/api/articles/${id}`, fetcher);
+
+  if (!article) return <p>Artikel Details werden geladen...</p>;
+  if (error) return <p>Fehler! keine Artikel Details gefunden.</p>;
+  const {
+    name,
+    price,
+    size,
+    gender,
+    status,
+    animal,
+    smoker,
+    description,
+    distance,
+    image,
+  } = article;
+  console.log(animal, smoker);
+  return (
+    <>
+      <StyledArticle>
+        <h3>Details zum Artikel: {name}</h3>
+        <Image
+          src={image}
+          alt={`Bild von ${name}`}
+          width={150}
+          height={200}
+        />
+        <ul>
+          <StyledList>Preis: {price} Euro</StyledList>
```

Comment:
Seems strange reading Styled LIST, but having a listitem and not a list ;-) Could be confusing for future Eugen ;-) Recommend changing the name.

Reply:
thanks for your great feedback Martin